libraries/bxdf/usd_preview_surface.mtlx

```diff
@@ -34,21 +50,52 @@
     <output name="rgba" type="color4" />
   </nodedef>
 
-<nodedef name="ND_UsdUVTexture_23" node="UsdUVTexture" nodegroup="texture2d" version="2.3" isdefaultversion="true">
-    <input name="file" type="filename" uniform="true" />
-    <input name="st" type="vector2" defaultgeomprop="UV0" />
-    <input name="wrapS" type="string" value="periodic" enum="black,clamp,periodic,mirror" uniform="true" />
-    <input name="wrapT" type="string" value="periodic" enum="black,clamp,periodic,mirror" uniform="true" />
-    <input name="fallback" type="color4" value="0, 0, 0, 1" />
-    <input name="scale" type="color4" value="1, 1, 1, 1" uniform="true" />
-    <input name="bias" type="color4" value="0, 0, 0, 0" uniform="true" />
+  <nodedef name="ND_UsdUVTexture_23" node="UsdUVTexture" nodegroup="texture2d" version="2.3" isdefaultversion="true"
+           doc="USD color texture reader version 2.3." >
+    <input name="file" type="filename" uniform="true" 
+           doc="Path to the color texture."/>
+    <input name="st" type="vector2" defaultgeomprop="UV0" 
+           doc="Texture coordinate to use to fetch this texture."/>
+    <input name="wrapS" type="string" value="periodic" enum="black,clamp,periodic,mirror" uniform="true" 
+           doc="Wrap mode when reading this texture."/>
+    <input name="wrapT" type="string" value="periodic" enum="black,clamp,periodic,mirror" uniform="true" 
+           doc="Wrap mode when reading this texture."/>           
+    <input name="fallback" type="color4" value="0, 0, 0, 1" 
+           doc="Fallback value used when texture can not be read. Default is ( 0, 0, 0, 1 )."/>
+    <input name="scale" type="color4" value="1, 1, 1, 1" uniform="true" 
+           doc="Scale to be applied to all components of the texture. Output is textureValue * scale + bias. Default is (1, 1, 1, 1)."/>
+    <input name="bias" type="color4" value="0, 0, 0, 0" uniform="true" 
+           doc="Bias to be applied to all components of the texture. Output is textureValue * scale + bias. Default is (0, 0, 0, 0)."/>
     <output name="r" type="float" />
     <output name="g" type="float" />
     <output name="b" type="float" />
     <output name="a" type="float" />
     <output name="rgb" type="color3" />
   </nodedef>
 
+  <nodedef name="ND_UsdUVTexture_23_vector3" node="UsdUvTexture" nodegroup="texture2d" version="2.3" isdefaultversion="true" 
```

Comment:
I still have some concerns about the proposed `vector3` variant of `UsdUvTexture`, and one initial issue is that it's taking different types than the `color3` variant for some inputs (e.g. scale and bias), making it no longer conformant to the Pixar specification.

Taking a step back, maybe we should break this proposal down into smaller pull requests, focusing the current one purely on the documentation improvements to `UsdPreviewSurface` and `UsdUvTexture`, which are clearly a worthwhile step to take.  Then we can consider new proposals for _type variations_ of these nodes, which will require additional discussion with Pixar and the community.